### PR TITLE
18284: digital business card

### PR DESCRIPTION
**Pull request by amanji**
*Issue #:* bcgov/entity#18284

**NOTE: do not merge until we determine whether this should be merged in a release branch.**

Target release: [Entities Sprint 19.3b Release - Digital Credentials](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvOTYzOTI)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/requirements.txt [hunk 1]
```diff
@@ -31,11 +32,15 @@ ecdsa==0.14.1
 expiringdict==1.1.4
 flask-jwt-oidc==0.3.0
 flask-restx==0.3.0
+git+https://github.com/bcgov/business-schemas.git@2.18.10#egg=registry_schemas
```
Comment (argush3) [inline legal-api/requirements.txt:35]: the business schema ref should be updated to 2.18.13

File: legal-api/src/legal_api/config.py [hunk 2]
```diff
@@ -144,9 +144,16 @@ class _Config():  # pylint: disable=too-few-public-methods
...
+    # Digital Credential configuration values
+    TRACTION_API_URL = os.getenv('TRACTION_API_URL')
+    TRACTION_API_TOKEN = os.getenv('TRACTION_API_TOKEN')
+    TRACTION_PUBLIC_SCHEMA_DID = os.getenv('TRACTION_PUBLIC_SCHEMA_DID')
+    TRACTION_PUBLIC_ISSUER_DID = os.getenv('TRACTION_PUBLIC_ISSUER_DID')
+
+    BUSINESS_SCHEMA_NAME = os.getenv('BUSINESS_SCHEMA_NAME')
+    BUSINESS_SCHEMA_VERSION = os.getenv('BUSINESS_SCHEMA_VERSION')
+    BUSINESS_SCHEMA_ID = os.getenv('BUSINESS_SCHEMA_ID')
+    BUSINESS_CRED_DEF_ID = os.getenv('BUSINESS_CRED_DEF_ID')
```
Comment (severinbeauvais) [inline legal-api/src/legal_api/config.py:156]: I'm not sure where these come from (for the BE) but probably need to work with Patrick Wei to configure these (in 1Password, I assume).

Are there different values for Dev/Test/Prod environments?

Also, where are these key values for running locally?

**Comment by amanji** [comment]
Closing as new PR created: https://github.com/bcgov/lear/pull/2260

**Comment by severinbeauvais** [comment]
You probably didn't need to close this PR. I think you could have clicked the Edit button at the top right and just changed the base.

**Pull request closed without merging**